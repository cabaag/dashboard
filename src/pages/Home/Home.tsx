import React from 'react';
import { Route, Switch } from 'react-router-dom';
import Layout from '../../components/Layout/Layout';
import Dashboard from '../Dashboard/Dashboard';
import User from '../User/User';

function Home() {
	return (
		<Layout>
			<Switch>
				<Route exact path='/'>
					<Dashboard />
				</Route>
				<Route exact path='/user'>
					<User />
				</Route>
			</Switch>
		</Layout>
	);
}

export default Home;
