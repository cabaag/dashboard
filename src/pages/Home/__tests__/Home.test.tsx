import { shallow } from 'enzyme';
import React from 'react';
import Home from '../Home';

describe('test Home component', () => {

		test('should render component', () => {
				expect(shallow(<Home />)).toMatchSnapshot();
		});

});
