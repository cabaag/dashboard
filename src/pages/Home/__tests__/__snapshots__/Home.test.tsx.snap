// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Home component should render component 1`] = `ShallowWrapper {}`;
